src/framework/components/element/element-drag-helper.js

```diff
@@ -215,7 +224,7 @@ class ElementDragHelper extends EventHandler {
 
         dragScale.x = 1 / dragScale.x;
         dragScale.y = 1 / dragScale.y;
-        dragScale.z = 1 / dragScale.z;
+        dragScale.z = 0;
```

Comment:
This is currently in PR https://github.com/playcanvas/engine/pull/4937

Reply:
merge that change it and resolve the comment if no longer relevant